discovery/service.go

```diff
@@ -239,33 +247,37 @@ func (d *Discovery) Stop() {
 func (d *Discovery) networkHandler() {
 	defer d.wg.Done()
 
-	var announcementBatch []*networkMsg
+	var announcementBatch []lnwire.Message
```

Comment:
Sure, that would be the case if it was a `lnwire.Message` from the get go, but this PR _changed_ the definition.